packages/esm-patient-vitals-app/src/config-schema.ts

```diff
@@ -58,6 +58,11 @@ export const configSchema = {
       _type: Type.UUID,
       _default: '67a71486-1a54-468f-ac3e-7091a9a79584',
     },
+    thresholdHours: {
```

Comment:
Nit:
Maybe we could rename this to be slightly more descriptive like - `vitalsOverdueThresholdHours`